node/src/tasks/msp_upload_file.rs

```diff
@@ -691,7 +691,8 @@ where
                 );
                 error!(target: LOG_TARGET, err_msg);
                 anyhow!(err_msg)
-            })?;
+            })?
+            .ok_or(anyhow!("MSP ID not found for bucket ID {:?}. This can happen if the MSP stopped storing the bucket while processing this storage request.", event.bucket_id))?;
```

Comment:
This is not really an error, is just a reason to skip the file, just like we do if `own_msp_id != msp_id_of_bucket_id`

Reply:
Yes that's a good point, will signal a warning log and skip the storage request in that case.